src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UICartService.java

```diff
@@ -110,7 +110,7 @@ public int removeProject(Long id) {
 	/**
 	 * Get the entire cart flushed out into {@link Project}s with their {@link Sample}s
 	 *
-	 * @return All proejcts and samples in the part
+	 * @return All projects and samples in the part
 	 */
 	public Map<Project, List<Sample>> getFullCart() {
```

Comment:
Ceate a dto which has a project and list of samples rather than returning a map

Reply:
This is used for legacy code and not to return to the UI.

Reply:
Oh ok. Disregard then